src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -455,6 +455,7 @@ trait Contexts { self: Analyzer =>
     def inReturnExpr                          = this(ReturnExpr)
     def inTypeConstructorAllowed              = this(TypeConstructorAllowed)
     def inAnnotation                          = this(TypingAnnotation)
+    def typingAnnotationCore              = this(TypingAnnotationCore)
```

Comment:
Alignment nerd alert. Also naive question, what does "core" mean? Is there a more usual term?

Reply:
It's the "fun" part, but here it just means the type. `typingAnnotationType`? `TypeOfAnnotation`?